### DISCUSSION TITLE
First release of this forked library

### DISCUSSION THREAD

**Discussion by pedrolcl** [EmbeddedSynth/sonivox-fork#1] [Announcements]
This is the first public release of this forked library, and of the original repository for what matters. It is unlikely that Google would release any day this code officially for any other platform than their Android, so here it is for everybody.

Why the strange release number **3.6.10.14**? It is the number that appears in the "eas.h" header [^1] as the library version identifier since the beginning. As the library was always private inside Android, this version number was never updated and was irrelevant data. Until now, that being a standalone library the version number really matters and will be updated for the next releases.

[^1]: https://github.com/pedrolcl/sonivox/blob/16df4aaa6cfdecd1f5280ab86112c7f5b40468d2/arm-wt-22k/host_src/eas.h#L45-L47

<hr /><em>This discussion was created from the release <a href='https://github.com/pedrolcl/sonivox/releases/tag/v3.6.10.14'>First release of this forked library</a>.</em>

**Comment by dvzrv** [comment]
Thanks for this being painless!
